src/map.react.js

```diff
@@ -78,6 +78,10 @@ const PROP_TYPES = {
     */
   height: PropTypes.number.isRequired,
   /**
+    * Override the default cursor.
+    */
+  cursor: PropTypes.string,
```

Comment:
If possible, add default value in DEFAULT_PROPS. Looking at the cursor method, don't we need two cursor props (GRAB/GRABBING)?